Old Apple Website features

warblo001 opened the pull request:
In this project we were tasked to create a clone from the Old Apple Website, using frontend technologies like gradients and backgrounds.
The project has the following features:
- Navigation Bar with gradients
- Backgrounds features on all the project
- Flexbox

warblo001 committed:
Added Markup Files and style folder

warblo001 committed:
Added links from Google Fonts and Font Awesome

warblo001 committed:
Added Navbar Markup

helciodev committed:
Finished markup

warblo001 committed:
Started navbar style

Added width to the navbar and centered content

helciodev committed:
Work on background position

warblo001 committed:
Styled Product Section

Changed images to background images in the Product Section

helciodev committed:
Styled hero-text

warblo001 committed:
Styled Navbar Section

Changed background color, added gradient

helciodev committed:
work on navbar height

warblo001 committed:
Styled footer

Styled paragraphs fonts and size and colors, changed links colors and aligned items

helciodev committed:
finish style

warblo001 committed:
Fixed linters issues

warblo001 committed:
Modified Live Server Demo link

javitocor commented on README.md:
```diff
@@ -1,2 +1,52 @@
+- Flexbox
+
+
+## Built With
+
+- HTML
+- CSS/SCSS
+
+
```
[OPTIONAL]
You can add SASS here, remember to add all the techs you use in the readme to highlight it

javitocor commented on README.md:
```diff
@@ -1,2 +1,52 @@
+
+## Authors
+
+😎 **Helcio André**
+
+- GitHub: [@helciodev](https://github.com/helciodev)
+- Twitter: [@helcio_bruno](https://twitter.com/helcio_bruno)
+
```
[OPTIONAL]

Add your LinkedIn link, recruiters will appreciate it

javitocor reviewed (changes requested):
# STATUS CHANGES REQUIRED ♻️ 

Great job with this project, team! 👍 
Your navbar is impressive, an amazing job with it looks like the original 👏 
Good and descriptive commits and a good readme file 💯 
Good use of backgrounds and gradients 🥇 
Your code is clean 🔝  and well structured 👍 
Linters are green 🟢 

However, we must do some changes in order to get the project approved.
Check my comments below 👇  apply the changes and request for a new code review ⏩ 

Good luck with it 😃 

## Changes

- As I said, the navbar is amazing, but the positioning it is not correct, check the difference with the original and kindly modify it. You can use background-size and position properties.
Original:
![Capture](https://user-images.githubusercontent.com/56809963/94947192-83c5e480-04dd-11eb-8969-a812110ceb6e.PNG)

Your project:
![Capture1](https://user-images.githubusercontent.com/56809963/94947269-a657fd80-04dd-11eb-8372-105c4a260ee9.PNG)

- Good use of flex, but one of the mandatory requirements is that you have to use float as well.   
       Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

helciodev committed:
Fixed height for banner and products

changed the font awesome icons for iconify icons adjusted background height to 503px and products to 200px

warblo001 committed:
Fixed footer position

Fixed search icon position, changed footer from flex, floated site-map to the right

warblo001 committed:
Fixed linters issues

warblo001 committed:
Fixed linters

Finished linters fixing

warblo001 committed:
Updated README.md

Modified Live Demo Link, LinkedIn link

helciodev committed:
fixed styles for hovering

arslanbisharat reviewed (approved):
## Approved
Thank you for making those changes requested by previous TSE. 
This project is approved.
Regards,
Arslan Bisharat
